scaaml/intro/generator.py

```diff
@@ -59,16 +59,16 @@ def create_dataset(filepattern,
         x = tf.concat(x, axis=0)
         y = tf.concat(y, axis=0)
 
-    cprint('[Generator]', 'yellow')
-    cprint('|-attack point:%s' % attack_point, 'blue')
-    cprint('|-attack byte:%s' % attack_byte, 'green')
-    cprint('|-num shards:%s' % num_shards, 'blue')
-    cprint('|-traces per shards:%s' % num_traces_per_shard, 'green')
-    cprint('|-y:%s' % str(y.shape), 'blue')
-    cprint('|-x:%s' % str(x.shape), 'green')
+    cprint("[Generator]", "yellow")
```

Comment:
As commented in PR #66 it would be better to ensure a consistent UX (and somehow give a purpose to color printing) by having helpers.

Reply:
https://github.com/google/scaaml/issues/67